### PR TITLE
Ensure that UTF-8 is used to encode/decode the data && Update dependencies && Update sbt plugins

### PR DESCRIPTION
**Pull request by guizmaii**
Hi @joan38,

I saw that your project was not up to date, so I decided to push you a PR.

While reading your code, I saw that you weren't using an explicit `Charset`. It can be a problem sometimes.
Be careful, with this Charset change, the new version of this lib may be non retro-compatible if Java chose another `Charset` during your program execution.

Cheers,
Jules

### CODE REVIEW

File: project/plugins.sbt [hunk 1]
```diff
@@ -1,6 +1,8 @@
 addSbtPlugin("io.get-coursier" % "sbt-coursier" % "1.0.2")
-addSbtPlugin("com.geirsson" % "sbt-scalafmt" % "1.5.0")
-addSbtPlugin("ch.epfl.scala" % "sbt-scalafix" % "0.5.10")
+addSbtPlugin("org.scalameta" % "sbt-scalafmt" % "2.0.2")
```
Comment (joan38) [inline project/plugins.sbt]: We can probably up that to `2.2.1`

Comment (guizmaii) [reply project/plugins.sbt]: Good call. Done

**Comment by guizmaii** [comment]
Thanks @joan38 🙂

**Comment by joan38** [comment]
@guizmaii Sorry for the huge delay but I just published `0.6` to maven central that contains your changes.

**Pull request merged**